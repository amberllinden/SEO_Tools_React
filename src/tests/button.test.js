import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/button';

test('Button component - some content with css-class "button-css"', () => {
    const tree = renderer
        .create(
            <Button
                onClickFuntion={() => 2}
                content="some content"
                styleClass="button-css"
            />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
