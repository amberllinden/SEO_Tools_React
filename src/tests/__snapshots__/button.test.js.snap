// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component - some content with css-class "button-css" 1`] = `
<button
  className="button-css"
  onClick={[Function]}
>
  some content
</button>
`;
